generateShopItems.js

```diff
@@ -1,11 +1,11 @@
-import { fetchData } from './getData.js'
+import { fetchData } from './api/apiCalls.js'
 import { ui } from './script.js'
-debugger
-function generateShopItem(data) {
+
+export function generateShopItem(data) {
 	const shopItemsList = data
 		.map((item) => {
 			return `
-  <li class="items-container__item">
+  <li data-id="${item.id}" class="items-container__item">
```

Comment:
Do you use this data-id field set on `li` anywhere?  
I reckon it should be enough to have it on the buttons..? 🤔

Reply:
Using those to remove item from list. I could probably figure out how to access data-id form button and use that, but was lazy to look at it :|
```
function deleteShopItemFromUi(itemToDeleteId) {
	let listItems = Array.from(ui.shopItemsList.getElementsByTagName('li'))
	listItems.map((item, index) => {
		if (item.dataset.id == itemToDeleteId) {
			ui.shopItemsList.removeChild(ui.shopItemsList.children[index])
		}
	})
}
```